language/class_spec.rb

```diff
@@ -285,6 +285,16 @@ def xyz
     }.should raise_error(TypeError)
   end
 
+  it "raises a TypeError when trying to extend non-Class" do
+    error_msg = /superclass must be a Class/
+    -> { Class.new("")              }.should raise_error(TypeError, error_msg)
+    -> { Class.new(1)               }.should raise_error(TypeError, error_msg)
+    -> { Class.new(:symbol)         }.should raise_error(TypeError, error_msg)
+    -> { Class.new(mock('o'))       }.should raise_error(TypeError, error_msg)
+    -> { Class.new(Module.new)      }.should raise_error(TypeError, error_msg)
+    -> { Class.new(BasicObject.new) }.should raise_error(TypeError, error_msg)
+  end
```

Comment:
These should use the `class` keyword

Reply:
oops, I forgot to change LOL....

Reply:
updated, now finally it should be done :)